docs/en/benchmark/profile_api_server.md

```diff
@@ -0,0 +1,86 @@
+# API Server Performance Test Method
+
+The way to profiling api_server performance is similar to the method for [profiling throughput](./profile_throughput.md). The difference is api_server should be launched successfully before testing.
+
+The evaluation script is `profile_restful_api.py`. Before running it, please install the lmdeploy precompiled package, download the evaluation script and the test dataset:
+
+```shell
+pip install lmdeploy
+git clone --depth=1 https://github.com/InternLM/lmdeploy
+wget https://huggingface.co/datasets/anon8231489123/ShareGPT_Vicuna_unfiltered/resolve/main/ShareGPT_V3_unfiltered_cleaned_split.json
+```
+
+During performance test, a specific model needs to be inputted. We recommend converting the model into turbomind format via `lmdeploy convert`, then proceed with testing.
+The reason is to conveniently adjust the parameters of the inference engine in order to achieve better performance, such as batch size (max_batch_size), K/V cache size (max_cache_entry_count), etc. For detailed explanations of these parameters, please refer to [here](../turbomind_config.md).
+
+In the following sections, we assume the model is in turbomind format.
+
+## Metrics
+
+LMDeploy records the performance metrics like first token latency, token throughput (tokens/s) and request throughput (RPM)
+
+`first_token_latency` is only reported in the case of streaming inference.
+
+The formula for calculating `token throughput` is:
+
+$$
+TokenThroughput=Number\\ of\\ generated\\ tokens/TotalTime
+$$
+
+And the formula for calculating `request throughput` is:
+$$
+RPM(request per minute)=Number\\ of\\ generated\\ tokens/TotalTime * 60
```

Comment:
This formula is not rendered as the exptected. There should be an empty line before `line 31 $$`
![image](https://github.com/InternLM/lmdeploy/assets/28671653/f5987c52-42c2-4874-aae3-3d30d8789536)